example/app.json

```diff
@@ -0,0 +1,30 @@
+{
+  "expo": {
+    "name": "example",
```

Comment:
Nitpick: maybe more description that just 'example'. Suggestion: `web3modal-reactnative-example`